src/styles/utilities/writingDirection.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Writing direction utility styles.
+ * note: writingDirection not available for Android, for Android you can use the Unicode.
+ *
+ * https://www.w3.org/International/questions/qa-bidi-unicode-controls
+ */
+export default {
+    rtl: {
+        writingDirection: 'ltr',
```

Comment:
This are both the same, not sure what the value of that is, of if this is a typo

Reply:
Done.